src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/request/TripPatternForDates.java

```diff
@@ -142,13 +144,40 @@ public RaptorTripPattern pattern() {
     }
 
     @Override
-    public boolean boardingPossibleAt(int stopPositionInPattern) {
+    public boolean boardingPossibleAt(
+            int stopPositionInPattern,
+            boolean wheelchairAccess
+    ) {
+
+        if (wheelchairAccess) {
+            var wheelchairPossible =
```

Comment:
In order to speed this up we probably want to add a method to TripPattern so it looks like this:

```
tripPattern.isWheelchairAccessible(stopPosInPattern)
```
In its implementation it should just look up a boolean in an array which is really, really fast. In this code path every millisecond counts.

This would also get rid of the duplication in the board/alight method.

Reply:
Good suggestion, I moved it there